tests/extensions/vb/Microsoft.DotNet.UpgradeAssistant.Extensions.VisualBasic.Tests/MyTypeTargetFrameworkSelectorFilterTests.cs

```diff
@@ -2,8 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Autofac.Extras.Moq;
+
 using Microsoft.DotNet.UpgradeAssistant.Extensions.VisualBasic;
+
 using Moq;
+
```

Comment:
Nit-pick - we probably don't need the extra lines here.

Reply:
This is an artifact of "Remove and Sort Usings" which groups using statements for me. I rather like it ;)

Reply:
Ok, works for me.